### ISSUE TITLE
Response Model Descriptions are incorrectly set and overwritten.

### ISSUE THREAD

**Issue by toshe** [ruby-grape/grape-swagger#578]
When an endpoint is generated in the swagger doc the `description` field of the response model is set incorrectly:

```ruby
@definitions[response_model][:description] = description_object(route)
```
The description_object(route) method returns the `description` of the route, which can be really long and is different for each route! 

So if you have an endpoint, say, getKittens:

```ruby
desc 'Get all kittens!', {
  nickname: 'getKittens',
  success: Entities::Kitten
  failure: [[401, 'KittenBitesError', Entities::BadKitten]]
  }
get '/kittens' do
   puts "meow"
end
```

Grape Swagger will define the new models Kitten and also BadKitten and will apply the same description in the the definition. Swagger generated doc:

```yml
definitions:
  kitten:
    type: object
    properties:
      fluffy:
        type: boolean
        description: Indicates if the kitten is fluffy or not!
    required:
      - fluffy
    description:  Get all kittens!
```

If you use the `KittenEntity` as a response somewhere else, then the model definition description will get overwritten once more with the description of the route.

Since the description of the route is in no way useful for the model description I suggest we simply replace it with a generic description:

```ruby
@definitions[response_model][:description] = "A #{response_model} model"
```

**Comment by LeFnord** [comment]
@toshe … yeap it makes sense to me, a PR would be nice 😉

**Comment by LeFnord** [comment]
closed via #804 https://github.com/ruby-grape/grape-swagger/pull/804
